src/components/Tooltip/Tooltip-test.js

```diff
@@ -162,19 +162,33 @@ describe('Tooltip', () => {
     });
   });
 
-  describe('getTriggerPosition', () => {
+  describe.skip('getTriggerPosition', () => {
```

Comment:
Is `.skip` here intentional?

Reply:
Whoops... I forgot to remove the skip. Will remove on next commit.